go.mod

```diff
@@ -3,85 +3,77 @@ module github.com/nutanix-cloud-native/ndb-operator
 go 1.20
 
 require (
-	github.com/google/uuid v1.1.2
+	github.com/google/uuid v1.3.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/ginkgo/v2 v2.1.4
-	github.com/onsi/gomega v1.19.0
-	github.com/stretchr/testify v1.7.0
-	go.uber.org/zap v1.21.0
-	k8s.io/api v0.25.0
-	k8s.io/apimachinery v0.25.0
-	k8s.io/client-go v0.25.0
-	sigs.k8s.io/controller-runtime v0.13.1
+	github.com/onsi/ginkgo/v2 v2.9.5
+	github.com/onsi/gomega v1.27.7
+	github.com/stretchr/testify v1.8.1
+	go.uber.org/zap v1.24.0
+	k8s.io/api v0.27.2
+	k8s.io/apimachinery v0.27.2
+	k8s.io/client-go v0.27.2
+	sigs.k8s.io/controller-runtime v0.15.1
```

Comment:
We are already upgrading 2 versions (v0.13.1 -> v0.15.1), I think we should not upgrade to v0.16.1 instantly. This may be more useful in the future when we upgrade to v0.16.1 but this is the error I received when I tried to upgrade to v0.16.1

Issues in moving controller-runtime to: v0.16.1


```
go vet ./...
# github.com/nutanix-cloud-native/ndb-operator/api/v1alpha1
vet: api/v1alpha1/webhook_suite_test.go:102:3: unknown field Host in struct literal of type manager.Options
# github.com/nutanix-cloud-native/ndb-operator
vet: ./main.go:78:3: unknown field MetricsBindAddress in struct literal of type manager.Options
make: *** [vet] Error 1
ndb-operator $
```